cropclassification/general.ini

```
@@ -39,6 +39,28 @@ reuse_last_run_dir = False
 # Do you want to reuse the config used in the run dir you want to reuse
 reuse_last_run_dir_config = False
 
+# The calc_periodic_mosaic_params section contains information to run a calc_marker task
+[calc_periodic_mosaic_params]
+
+# The days of the period to download the mosaics
+days_per_period = 7
+# The start date of the period we want to download the mosaic
+start_date_str = MUST_OVERRIDE
+# The end date of the period we want to download the mosaic
+# Also supports placeholder {now}, to use the current date
+end_date_str = MUST_OVERRIDE
+# The days before the end_date
+# If not None, subtract end_date_subtract_days from end_date
+end_date_subtract_days = 3
+# The path to download the mosaics
+dest_image_data_dir = /tmp/satellite_periodic
+# The sensor
+sensors = s2-agri
+# The CRS of the roi
+roi_crs = 31370
+# Simulate, nothing is being downloaded
+simulate = MUST_OVERRIDE
```

```suggestion
simulate = False
```
I think it is more logical to use False as default here?

It is typically only for testing purposes that simulate is relevant?

---

yes indeed